### PR TITLE
Huawei VRF BGP_Peers update

### PR DESCRIPTION
**Pull request by PipoCanaja**
- MIB update
- BGP Peer description support, including test data
- FIX for bgpPeersTable because of different indexes
- FIX for sql update of existing peer when VRF is NULL

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/discovery/bgp-peers/vrp.inc.php [hunk 1]
```diff
@@ -42,18 +42,27 @@
             $map_vrf['byId'][$vrf['vrf_id']]['vrf_name'] = $vrf['vrf_name'];
             $map_vrf['byName'][$vrf['vrf_name']]['vrf_id'] = $vrf['vrf_id'];
         }
-        $bgpPeersCache = snmpwalk_cache_oid($device, 'hwBgpPeers', [], 'HUAWEI-BGP-VPN-MIB');
+
+        $bgpPeersCache = snmpwalk_cache_oid($device, 'hwBgpPeerAddrFamilyTable', $bgpPeersCache, 'HUAWEI-BGP-VPN-MIB');
+        $bgpPeersCache = snmpwalk_cache_oid($device, 'hwBgpPeerTable', $bgpPeersCache, 'HUAWEI-BGP-VPN-MIB');
+        $bgpPeersCache = snmpwalk_cache_oid($device, 'hwBgpPeerRouteTable', $bgpPeersCache, 'HUAWEI-BGP-VPN-MIB');
+        $bgpPeersCache = snmpwalk_cache_oid($device, 'hwBgpPeerSessionTable', $bgpPeersCache, 'HUAWEI-BGP-VPN-MIB');
+
```
Comment (PipoCanaja) [inline includes/discovery/bgp-peers/vrp.inc.php:50]: This is needed cause the parent OID (which was polled before) may include data with different indexing scheme, and it breaks the logic.
bgpPeersDesc is one of those.

File: includes/discovery/bgp-peers/vrp.inc.php [hunk 2]
```diff
@@ -105,7 +119,7 @@
                     echo '+';
                     $vrp_bgp_peer_count++;
                 } else {
-                    dbUpdate(['bgpPeerRemoteAs' => $value['hwBgpPeerRemoteAs'], 'astext' => $astext], 'bgpPeers', 'device_id = ? AND bgpPeerIdentifier = ? AND vrf_id = ?', [$device['device_id'], $address, $vrfId]);
+                    dbUpdate(['bgpPeerDescr' => $value['bgpPeerDescr'], 'bgpPeerRemoteAs' => $value['hwBgpPeerRemoteAs'], 'astext' => $astext], 'bgpPeers', 'device_id = ? AND bgpPeerIdentifier = ? ' . $checkVrf, [$device['device_id'], $address, $vrfId]);
```
Comment (PipoCanaja) [inline includes/discovery/bgp-peers/vrp.inc.php:122]: FIX Fixes updates when vrfId is NULL
ADD Adds description update

**Comment by PipoCanaja** [comment]
Hello @librenms/reviewers 
Wondering if this PR could get reviewed, considering the half month release will be around soon and I would love to see it in
Thanx

**Review by murrant — APPROVED** [review]

**Comment by murrant** [comment]
Looks good, sorry we didn't make it for the last release.

**Pull request merged**